kyuubi-server/src/main/scala/org/apache/kyuubi/engine/hive/HiveProcessBuilder.scala

```diff
@@ -39,6 +39,8 @@ class HiveProcessBuilder(
 
   private val hiveHome: String = getEngineHome("hive")
 
+  private val HIVE_HADOOP_CLASSPATH: String = "HIVE_HADOOP_CLASSPATH"
```

Comment:
nit: unnecessary val?

Reply:
It's usefull. I change all "HIVE_HADOOP_CLASSPATH" literal to this constant. In the future, we could move all similar constants to a scala file.

The following code uses this constant.
https://github.com/deadwind4/incubator-kyuubi/blob/9f9fcdac5b6f77eb03e20d6e41f95b03a1dc9568/kyuubi-server/src/main/scala/org/apache/kyuubi/engine/hive/HiveProcessBuilder.scala#L82-L95